sn_networking/src/lib.rs

```diff
@@ -671,7 +670,7 @@ impl Network {
                 Ok(res) => format!("{res}"),
                 Err(err) => format!("{err:?}"),
             };
-            debug!("Got response for the req: {req:?}, res: {res_string}");
```

Comment:
I think here we need to `log` some ID stuff of the `req` , so that we can trace it down.